src/Compatibility/Microsoft.DotNet.PackageValidation/sdk/Sdk.props

```diff
@@ -0,0 +1,4 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project>
+  <Import Project="..\build\Microsoft.DotNet.PackageValidation.props" />
```

Comment:
NIT: If this package is meant to be an SDK, should it be called Microsoft.DotNet.PackageValidation.Sdk instead to match the rest of our sdks?

Reply:
Those files are intentionally names as the PackageId so that the packages could also be consumed via a PackageReference (i.e. useful if the sdk is imported by default and you want to override its version).